core/src/main/scala/cats/data/IndexedReaderWriterStateT.scala

```diff
@@ -462,6 +462,12 @@ private[data] sealed abstract class IRWSTInstances extends IRWSTInstances1 {
       implicit def L: Monoid[L] = L0
     }
 
+  implicit def catsDataDefer[F[_], E, L, SA, SB](implicit F: Defer[F]): Defer[IndexedReaderWriterStateT[F, E, L, SA, SB, ?]] =
```

Comment:
totally trivial nit: the naming convention has a `ForIRWST` suffix. Although I don't think it matters much since there is no risk of naming conflict here.

Reply:
yep, that was a goof up.